web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -183,6 +189,7 @@ export default defineComponent({
         await harmonizerApi.init(r, schema, activeTemplate.value.schemaClass, goldEcosystemTree);
         await nextTick();
         harmonizerApi.loadData(activeTemplateData.value);
+        harmonizerApi.addBeforeChangeHook(onBeforeChange);
```

Comment:
I think you might also need to add this in the `changeTemplate` function. It looks to me like if I change tabs in the data harmonizer step I stop seeing the new element change between loading and verifying.

Reply:
Good catch!